src/dotnet/GeneratorAgent/src/Azure.Tools.GeneratorAgent/Interfaces/ILoggerService.cs

```diff
@@ -0,0 +1,10 @@
+namespace Azure.Tools.GeneratorAgent.Interfaces
+{
+    public interface ILoggerService
```

Comment:
Can we just use the built-in ILogger, e.g.
```c#
serviceCollection.AddLogging(builder =>builder.AddConsole());
```

Then we can inject an ILogger into our service constructor and use that to log to the console.

Reply:
+1 to using `ILogger` and its infrastructure.   That saves us from having to run it all ourselves and lets us interop with the wider ecosystem.